@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -1079,8 +948,205 @@ export interface TextTechnique extends BaseTechnique {
      * Fading time for labels in seconds.
      */
     textFadeTime?: number;
+    /**
+     * Clip the line outside the tile if `true`.
+     */
+    clipping?: boolean;
+    /**
+     * The name of the property containing the text.
+     */
+    textLabel?: string;
+}
+
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    BaseStandardTechniqueParams,
+    BaseTechniqueParams,
+    BasicExtrudedLineTechniqueParams,
+    DashedLineTechniqueParams,
+    ExtrudedPolygonTechniqueParams,
+    FillTechniqueParams,
+    LineTechniqueParams,
+    MarkerTechniqueParams,
+    PointTechniqueParams,
+    SegmentsTechniqueParams,
+    ShaderTechniqueParams,
+    SolidLineTechniqueParams,
+    StandardExtrudedLineTechniqueParams,
+    StandardTexturedTechniqueParams,
+    TextTechniqueParams
+} from "./Techniques";
+
+export interface SquaresTechnique extends PointTechniqueParams {
+    /**
+     * Name of technique
+     */
+    name: "squares";
+}
+
+export interface CirclesTechnique extends PointTechniqueParams {
+    /**
+     * Name of technique. Is used in the theme file.
+     */
+    name: "circles";
 }
 
+/**
+ * Declares a geometry as a point of interest (POI).
+ */
+export interface PoiTechnique extends MarkerTechniqueParams {
+    /**
+     * Name of technique. Is used in the theme file.
+     */
+    name: "labeled-icon";
+}
+
+/**
+ * Declares a geometry as a line marker.
+ */
+export interface LineMarkerTechnique extends MarkerTechniqueParams {
+    /**
+     * Name of technique. Is used in the theme file.
+     */
+    name: "line-marker";
+    /**
+     * Minimal distance between markers in screen pixels.
+     */
+    minDistance?: number;
```

Comment:
Kind of weird that this is a member or ``LineMarkerTechnique`` and not of ``MarkerTechniqueParams`` (specially when this is the only technique where this happens). Wouldn't it make more sense to store this value inside of ``MarkerTechniqueParams``, or create ``LineMarkerTechniqueParams extends MarkerTechniqueParams``?